website/docs/docs/on-premises/releases/1.26.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 1.26.0
+---
+
+## Release date
+
+07 August 2023
+
+## Features
+
+* Support application/problem+json error response format for all non-200 responses
+* Major version upgrade of AJV from v6 to v8 and Improve support for allOf, anyOf, oneOf, including polymorphic inheritance
```

Comment:
Could you please link to this page: https://docs.pactflow.io/docs/bi-directional-contract-testing/contracts/oas/changelog

This should help them discover the changes  in that release.